crates/normalize-path/src/lib.rs

```diff
@@ -1,13 +1,35 @@
-//! Shamelessly adapted from:
-//! https://github.com/rust-lang/cargo/blob/fede83ccf973457de319ba6fa0e36ead454d2e20/src/cargo/util/paths.rs#L61
+//! Normalizes paths similarly to canonicalize, but without performing I/O.
+//!
+//! This is like Python's `os.path.normpath`.
+//!
+//! Initially adapted from [Cargo's implementation][cargo-paths].
+//!
+//! [cargo-paths]: https://github.com/rust-lang/cargo/blob/fede83ccf973457de319ba6fa0e36ead454d2e20/src/cargo/util/paths.rs#L61
+//!
+//! # Example
+//!
+//! ```
+//! use normalize_path::NormalizePath;
+//!
```

Comment:
```suggestion
//! use std::path::Path;
//!
```

This will fix the unit test CI.